### PR TITLE
Lift him (PR that makes it so you can be lifted)

### PR DESCRIPTION
**Pull request by ReturnToZender**
## About The Pull Request

Source inspiration: https://github.com/Bubberstation/Bubberstation/pull/1042
This PR adds an optional 1-point positive quirk that allows you to be picked up. This has no real functional difference from the fireman's carry.

Naturally, this trait is mutually exclusive with the oversized quirk.

## Why It's Good For The Game

https://github.com/Bubberstation/Bubberstation/assets/110273561/795a66fb-2073-457e-9aab-333fae67117b

## Proof Of Testing
image

## Changelog

:cl: ReturnToZender (quirk code), TeshariEnjoer (original PR)
add: New positive trait, featherweight. Allows you to be picked up instead of fireman's carried. This dipshit yours?
/:cl:

### CODE REVIEW

**Comment by nevimer** [comment]
remind me to testmerge

File: modular_zubbers/code/datums/components/datums/riding_mob.dm [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#define DEFAULT_IN_HAND_OFFSET_X 3
+#define DEFAULT_IN_HAND_OFFSET_Y 0
+
+/datum/component/riding/creature/human
+	var/obj/item/bodypart/used_hand
+
+/datum/component/riding/creature/human/Initialize(mob/living/riding_mob, force, ride_check_flags, potion_boost)
+	. = ..()
+	var/mob/living/carbon/human/human_parent = parent
```
Comment (Cyprex) [inline modular_zubbers/code/datums/components/datums/riding_mob.dm:9]: rename this to steed/ridden/hunan_ridden or something of the like- it's not 100% clear that this is the person being ridden

Comment (ReturnToZender) [reply modular_zubbers/code/datums/components/datums/riding_mob.dm:9]: will do

File: modular_zubbers/code/datums/components/datums/riding_mob.dm [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	if(ride_check_flags & CARRIER_NEEDS_ARM && HAS_TRAIT(riding_mob, TRAIT_CAN_BE_PICKED_UP))
+		human_parent.buckle_lying = 0
```
Comment (nikothedude) [inline modular_zubbers/code/datums/components/datums/riding_mob.dm:11]: YOU DARE TO USE 0 IN PLACE OF FALSE

ITS 2024 YOU MOTH

File: modular_zubbers/code/game/traits/positive.dm [hunk 3]
```diff
@@ -0,0 +1,11 @@
+/datum/quirk/featherweight
+	name = "Featherweight"
+	desc = "Your body is lighter than others'. You can be picked up by the scruff or collar with ease."
+	value = 1
```
Comment (nikothedude) [inline modular_zubbers/code/game/traits/positive.dm:4]: why does this have a value at all

Comment (ReturnToZender) [reply modular_zubbers/code/game/traits/positive.dm:4]: I originally thought it was a _slight_ upgrade to the fireman's carry for someone else, but afterwards, I realized it was just the same.

Kinda didn't update it

**Pull request merged**